types/chrome/index.d.ts

```diff
@@ -10019,14 +10028,14 @@ declare namespace chrome.vpnProvider {
 declare namespace chrome.wallpaper {
     export interface WallpaperDetails {
         /** Optional. The jpeg or png encoded wallpaper image. */
-        data?: any;
+        data?: ArrayBuffer;
         /** Optional. The URL of the wallpaper to be set. */
         url?: string | undefined;
         /**
          * The supported wallpaper layouts.
          * One of: "STRETCH", "CENTER", or "CENTER_CROPPED"
          */
-        layout: string;
+        layout: "STRETCH" | "CENTER" | "CENTER_CROPPED";
```

Comment:
See comment above and docs: https://developer.chrome.com/docs/extensions/reference/wallpaper/#type-WallpaperLayout

Reply:
Since the `exactOptionalPropertyTypes` typescript flag is set, undefined should be explicitly declared.
```suggestion
        layout: "STRETCH" | "CENTER" | "CENTER_CROPPED" | undefined;
```